### ISSUE TITLE
Display kubernetes manifest (yaml) of kubernetes objects running in Podman 

### ISSUE THREAD

**Issue by slemeur** [containers/podman-desktop#518]
## Description

As a user of Podman Desktop, when I'm running a kubernetes objects (Pod or other), I'd like to be able to see (with a text editor) the YAMLs which define those objects.

Two sample use cases:
- I started running a Pod from existing YAML files, and I'd like to be able to see those YAMLs for each of kubernetes objects (Pods, Deployment, PVC, Services...) along with my Pod, directly within Podman Desktop.
- I have used Podman capabilities to run a Pod. Podman allows to generate the Kubernetes yaml files corresponding to that Pod. Even without "explicitely generating the YAML" , I'd still like to see the kubernetes yaml files. 

Basically, it means that we need a detail page for a Pod. One tab of that page will display all the YAMLs corresponding to the Pod.

**Comment by benoitf** [comment]
could you clarify the difference between 'visualize' and 'see'
What is the 'manifest yaml' ? Kubernetes yaml ?

Could you explain more  (for example the meaning of 'visualize each object') and 'visualize what has been generated'
